### DISCUSSION TITLE
Proposition for a new Firefox logo SVG file using the updated design

### DISCUSSION THREAD

**Discussion by Nicryc** [jellyfin/jellyfin-web#3766] [Polls]
Hello,

I'd like to propose a new SVG file for the Firefox logo with the updated design. I've made 4 different versions and I'd like to ask you which one do you prefer before making a pull request.

As a reminder, this SVG file is intended to be used in the 'Devices' tab of Jellyfin's settings when someone is connected to your server using Firefox.
![jellyfin_devices_screenshot](https://user-images.githubusercontent.com/15157120/178981615-90089749-bed2-4759-b77d-e73dbde7d176.png)

1. **Without the globe**
<img src="https://user-images.githubusercontent.com/15157120/178979582-4e052140-b127-45bf-bad1-c2e57d7429dc.png" alt="without_globe" width="200"/>

2. **With globe border only**
<img src="https://user-images.githubusercontent.com/15157120/178979946-7d5a7f4c-c403-4259-97c6-566076b96ccc.png" alt="with_globe_border_only" width="200"/>

3. **With the globe**
<img src="https://user-images.githubusercontent.com/15157120/178980086-aeac58fa-5f56-4246-857d-786b8dd6c888.png" alt="with_globe" width="200"/>

4. **With the globe but hand reworked** (notice the difference on the globe near the tail)
<img src="https://user-images.githubusercontent.com/15157120/178980198-96e5576e-a0dd-49a5-a0e0-79f01dc28a30.png" alt="with_globe_hand_reworked" width="200"/>

Thank you to vote in the poll or add a comment.

**Comment by thornbill** [comment]
I don't have a strong opinion on this. Originally we used [Simple Icons](https://simpleicons.org) as a source for these. It looks like they do have a new Firefox logo available as well that uses the globe border.
